operator/cmd/mesh/profile-diff.go

```diff
@@ -39,11 +39,11 @@ func profileDiffCmd(rootArgs *rootArgs, pfArgs *profileDiffArgs) *cobra.Command
 		Use:   "diff <file1.yaml> <file2.yaml>",
 		Short: "Diffs two Istio configuration profiles",
 		Long:  "The diff subcommand displays the differences between two Istio configuration profiles.",
-		Example: `  # Profile diff by providing yaml files
-  istioctl profile diff manifests/profiles/default.yaml manifests/profiles/demo.yaml
+		Example: `  # Profile diff by providing profile names
+  istioctl profile diff default demo
 
-  # Profile diff by providing a profile name
-  istioctl profile diff default demo`,
+  # Profile diff by providing manifests path and profile names
+  istioctl profile diff default demo -d manifests/`,
```

Comment:
this is needed only for dev builds

Reply:
not only for dev build, especially useful if customized chart is specified.